commodore/catalog.py

```diff
@@ -84,14 +102,20 @@ def update_catalog(cfg, target_name, repo):
         click.echo(textwrap.indent(commit_message, '   '))
     if changed:
         if not cfg.local:
+            if cfg.interactive:
+                if yes_or_no('Should the push be done?'):
```

Comment:
You can use `click.confirm()` to ask yes/no questions: https://click.palletsprojects.com/en/1.x-maintenance/prompts.html#confirmation-prompts